### ISSUE TITLE
Absence of password-store-otp

### ISSUE THREAD

**Issue by DamienCassou** [emacsmirror/.github#2]
password-store-otp has been available in melpa for 3 days now: https://melpa.org/#/password-store-otp and https://github.com/melpa/melpa/pull/4983. Is it normal that the package is still not part of the emacsmirror? I'm not in a hurry, I just wandered if there was a bug.

**Comment by tarsius** [comment]
> available in melpa for 3 days now [...] Is it normal that the package is still not part of the emacsmirror?

Yes, adding new packages is only semi-automated and I don't do it every day.

However the current situation is a bit more complicated. `password-store-otp` actually is available from the Emacsmirror, but the database in the `master` branch of `epkgs` does not contain an entry. Two months ago I made some far reaching changes including to the database schemata and since then I have only updated the `next` branch. I plan to release updates to the relevant packages and to fast-forward `epkg`'s `master` to `next` very soon.

It's unfortunate that the stable version of the database is lagging behind so far right now. I didn't want this situation to last for more than maybe two weeks (for testing purposes), but so many things got in the way I just didn't find the time to finish things up. Today I am trying to make a new magit release, after that I will start working toward new borg/epkg/... releases again. Not sure I will be able to actually make new releases this week though -- I have to concentrate on the magit campaign. But I do intend to create these releases before the campaign is over.
